plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Wrapper.java

```diff
@@ -333,6 +388,26 @@ public ObjectMetadata getObject(GetObjectRequest getObjectRequest, File destinat
     }
 
     @Override
+    public String getObjectAsString(final String bucketName, final String key) throws AmazonServiceException, AmazonClientException {
+        return delegate.getObjectAsString(bucketName, key);
+    }
+
+    @Override
+    public GetObjectTaggingResult getObjectTagging(final GetObjectTaggingRequest getObjectTaggingRequest) {
+        return delegate.getObjectTagging(getObjectTaggingRequest);
+    }
+
+    @Override
+    public SetObjectTaggingResult setObjectTagging(final SetObjectTaggingRequest setObjectTaggingRequest) {
```

Comment:
Are all these new overrides necessary because of the upgrade? We don't use any of these methods, so if we can avoid adding overrides I think that would be preferable.

Reply:
AFAIR it's necessary.